### ISSUE TITLE
Translating PSR-7 to Brazilian Portuguese

### ISSUE THREAD

**Issue by marcelgsantos** [php-fig/fig-standards#719]
Hello!

I started some time ago to translate PSR-7 to Brazilian Portuguese but today I noticed that translations were removed from this repository. What should I do now?

Thanks,

Marcel dos Santos

**Comment by henriquemoody** [comment]
Hey, @marcelgsantos.

Not only Brazilian Portuguese translations were closed, see https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/php-fig/6xiVurQVwa0/Qo0kJhccCgAJ.

**Comment by henriquemoody** [comment]
However, I think anyone can fork the repository and publish a *non official* translation.

**Comment by marcelgsantos** [comment]
Hey @henriquemoody!

I was reading many threads in PHP-FIG Google Group and some pull requests about it. I understood the reason but I thought too radical!

There is a lot of people that don't understand English very well and technical translations using Google Translate is a mess. Not translate these documents, we are limiting them only to few people.

I decided to translate PSR-7 because I saw that Brazilian community was involved in translate these documents like you.

Thank you for reply! :smile: 

Marcel dos Santos
